INSTALL.md

```diff
@@ -179,7 +179,7 @@ the FFTW3 threading library libfftw3_threads (or libfftw3_omp) is required.
 
 - Specify OFFLOAD_CC (e.g. `OFFLOAD_CC = nvcc`) and
   OFFLOAD_FLAGS (e.g. `OFFLOAD_FLAGS = -O3 -g -w --std=c++11`) variables.
-- Use the `-D__DBCSR_ACC` to enable accelerator support for matrix multiplications.
+- Use the `-D__DBCSR_ACC` and `USE_ACCEL=cuda` to enable accelerator support for matrix multiplications.
```

Comment:
Formatting : need to the line after for. That's why the CI fails.

Reply:
Added a newline after for to break up the sentence into two.

Reply:
Unfortunately, (markdownlint)[https://github.com/DavidAnson/markdownlint] is still complaining about line length and trailing spaces. You can run this checks yourself via `make pretty`.

Reply:
make pretty was useful. Removed all those trailing whitespaces.